.travis.yml

```diff
@@ -18,9 +32,9 @@ script:
   # linting
   - pylint texar/ examples/
   - flake8 texar/ examples/
-  # type-checking
-  - mypy .
-  - for dir in `echo examples/**/`; do mypy $dir; done
+  # type-checking (only for PyTorch version 1.2.0)
+  - "if [[ $TORCH_VER != '1.0.1' ]]; then mypy .; fi"
```

Comment:
I am not very familiar with this part. Does this mean we only run `mypy` for `torch 1.1.0` and `torch 1.2.0`? If that is the case, why do we need to do this?

Reply:
Yes. This is because PyTorch 1.0.1 has almost no stubs (or wacky ones), so type check throws a bunch of errors. I don't think it's necessary to put too much effort  into resolving those, plus it (probably) shortens the CI run time.